attest-data/src/messages.rs

```diff
@@ -2,18 +2,21 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-use crate::NONCE_SIZE;
+use crate::{NONCE_SIZE, SHA3_256_DIGEST_SIZE};
+use hubpack::error::Error as HubpackError;
 use hubpack::SerializedSize;
 use serde::{de::DeserializeOwned, Deserialize, Serialize};
 
-use hubpack::error::Error as HubpackError;
-
 /// Magic value for [`Header::magic`]
 pub const ATTEST_MAGIC: u32 = 0xA77E5700;
 
-/// Right now `Attest` is the only command that takes data (nonce)
+/// Right now `Attest` and `TqSign` are the only commands that take data
+/// argumenets. They happen to be the same length but to be extra cautious
+/// add a static assertion.
 pub const MAX_DATA_LEN: usize = NONCE_SIZE;
 
+static_assertions::const_assert!(SHA3_256_DIGEST_SIZE == 32);
```

Comment:
This feels weird. Isn't this a tautology?

Reply:
I had some massive brain farts while working on the code and intended this to be related to the size of the hash passed to `TqSign` (I wasted several hours thinking the size of sha256 was 64 bytes..). You're right this is not actually useful so I'll remove it.